scripts/prune_lm_dir.py

```diff
@@ -21,6 +22,15 @@
                     'prune*X, with X <= 1.0, tells it to prune with X times the threshold '
                     'specified with the --threshold option.  EM specifies one iteration of '
                     'E-M on the model. ')
+parser.add_argument("--target-size", type=int, default=0,
+                    help="Target size of final LM after pruning. "
+                    "If setting this to a positive value, the --steps would be "
+                    "ignored and a few steps may be worked out util the size "
+                    "of pruned LM match the target-size.")
+parser.add_argument("--tolerance", type=float, default=0.05,
+                    help="Tolerance of actual size of final LM and the target-size. ")
+parser.add_argument("--max-attempt", type=int, default=20,
+                    help="Max attempts to find target size LM. ")
```

Comment:
There is a log at the end of prune_lm_dir.py said ".... log-prob changes per iter were ....", then it prints the log changes for every step(including prune and EM). So i thought the word a  iteration means a step here. But this 'attempt' here i mean a Prune step followed by 3 EM step. They are different, could you provide another name instead of iter?